main.tf

```diff
@@ -0,0 +1,509 @@
+# Copyright 2024 Canonical Ltd.
```

Comment:
It feels a bit odd to have this module in the main folder while other SD-Core modules are stored under `modules`. Especially in the situation where this module doesn't reuse `sdcore-control-plane-k8s` and `sdcore-user-plane-k8s`.
How about we move it together with the rest of the modules?